### ISSUE TITLE
Dashboard Slow?

### ISSUE THREAD

**Issue by Jiriteach** [alexjustesen/speedtest-tracker#92]
Any reason why the entire dashboard would be slow? Seems that the http server is noticably slow to respond. 
Running this on a Raspberry Pi 4. Speedtests are fine - super fast with no issues but browing the dashboard is slow.

**Comment by alexjustesen** [comment]
@Jiriteach can you inspect the page with dev tools and give me an idea of what might be running slow and also provide some specifics around what "slow" means?

**Comment by Jiriteach** [comment]
@alexjustesen  - Sure. Looks like the filament notifications. ~ 2.5 secs could the cause.
See the screenshot.

<img width="1552" alt="Screen Shot 2022-10-29 at 06 32 34" src="https://user-images.githubusercontent.com/2116446/198697931-3bb06a52-7564-4e65-92ce-f44b1985ab61.png">

**Comment by alexjustesen** [comment]
Interesting, so I know I have notification polling set to 5s. I wonder if this is causing the issue. I'll look into it.

**Comment by alexjustesen** [comment]
I wasn't able to reproduce the performance issue in this case, I'm seeing 32ms on my local database, I'd check the performance of the database you're using to make sure it's not causing the slow response when checking for a notification.

**Comment by alexjustesen** [comment]
I'll keep monitoring this but closing the issue for now

**Comment by alexjustesen** [comment]
I suggest listening to https://share.transistor.fm/s/600fbf9b. 

Tldr: improvements coming in Filament v4 (the admin panel I'm using).
